homeassistant/components/switchbot_cloud/light.py

```diff
@@ -0,0 +1,115 @@
+"""Support for the Switchbot Light."""
+
+from typing import Any
+
+from switchbot_api import CommonCommands, Device, RGBWWLightCommands, SwitchBotAPI
+
+from homeassistant.components.light import ColorMode, LightEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddConfigEntryEntitiesCallback
+
+from . import SwitchbotCloudData, SwitchBotCoordinator
+from .const import DOMAIN
+from .entity import SwitchBotCloudEntity
+
+
+def value_map_brightness(value: int) -> int:
+    """Return value for brightness map."""
+    return int(value / 255 * 100)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config: ConfigEntry,
+    async_add_entities: AddConfigEntryEntitiesCallback,
+) -> None:
+    """Set up SwitchBot Cloud entry."""
+    data: SwitchbotCloudData = hass.data[DOMAIN][config.entry_id]
+    async_add_entities(
+        SwitchBotCloudLight(data.api, device, coordinator)
+        for device, coordinator in data.devices.lights
+    )
+
+
+class SwitchBotCloudLight(SwitchBotCloudEntity, LightEntity):
+    """Representation of a SwitchBot Battery Circulator Fan."""
+
+    _attr_max_color_temp_kelvin = 6500
+    _attr_min_color_temp_kelvin = 2700
+
+    _attr_is_on: bool | None = None
+
+    def __init__(
+        self,
+        api: SwitchBotAPI,
+        device: Device,
+        coordinator: SwitchBotCoordinator,
+    ) -> None:
+        """Entity init."""
+        super().__init__(api, device, coordinator)
+        if device.device_type not in "Strip Light":
```

Comment:
I think this is incorrect, You have `Strip Light` and `Strip Light 3` and only one is matched, and if that is expected we should use `==`

Reply:
What about Strip Light 3?